### ISSUE TITLE
Allow explicit declaration of @fake and @examples directives in schema

### ISSUE THREAD

**Issue by richard-ejem** [APIs-guru/graphql-faker#75]
I use an explicit declaration of `@fake` and `@examples` directive like this:

```
directive @examples(
	values: [String!]!
) on FIELD_DEFINITION | SCALAR

directive @fake(
	type: GraphQLFakerFakeTypes!
) on FIELD_DEFINITION | SCALAR
```

Otherwise, IDEA's GraphQL plugin complains that these directives are unknown.

However, GraphQL faker server returns this error in version 2.0.0-rc2:

`{"errors":[{"message":"There can be only one directive named \"fake\".\n\nThere can be only one directive named \"examples\"."}]}`

It worked properly in GraphQL faker version 1.9.2 with these declarations. I think they should be allowed to be present in the schema.

**Comment by IvanGoncharov** [comment]
@richard-ejem Thanks for detail report 👍 
Fixed in `v2.0.0-rc.4` 📦 
Can you please test it?

**Comment by richard-ejem** [comment]
@IvanGoncharov great, it seems to work now, thanks for quick fix 👍
